keyboards/honeycomb/matrix.c

```diff
@@ -0,0 +1,202 @@
+/*
+Copyright 2012 Jun Wako
+Copyright 2014 Jack Humbert
+Copyright 2019 @filoxo
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#include <stdint.h>
+#include <stdbool.h>
+#if defined(__AVR__)
+#include <avr/io.h>
+#endif
+#include "wait.h"
+#include "print.h"
+#include "debug.h"
+#include "util.h"
+#include "matrix.h"
+#include "timer.h"
+#include "honeycomb.h"
+#include "pointing_device.h"
+#include "report.h"
+
+#if (MATRIX_COLS <= 8)
+#    define print_matrix_header()  print("\nr/c 01234567\n")
```

Comment:
```suggestion
  #define PRINT_MATRIX_HEADER()  print("\nr/c 01234567\n")
```
Use `CONSTANT_CASE` for preprocessor constant/macro names. Don't add spaces between `#` and the preprocessor directive; indent the whole line instead (by 2, not 4). Same applies to the rest of the lines below.

Reply:
This was the only suggestion that I could not incorporate. These are not macros and are required to be named exactly, so they need to remain lowercase. There are a few other keyboards with this exact same code ([chimera boards]( https://github.com/qmk/qmk_firmware/blob/master/keyboards/chimera_ls/matrix.c#L31-L34), for example) so I wonder if the original author copied them directly.